src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -45,61 +40,39 @@ public ElevatorCommandsBuilder getCommandsBuilder() {
 
 	protected void setPower(double power) {
 		frontMotor.setPower(power);
-		backMotor.setPower(power);
 	}
 
 	protected void setVoltage(double voltage) {
 		frontMotor.applyDoubleRequest(voltageRequest.withSetPoint(voltage));
-		backMotor.applyDoubleRequest(voltageRequest.withSetPoint(voltage));
 	}
 
 	protected void stop() {
 		frontMotor.stop();
-		backMotor.stop();
 	}
 
 	public void setBrake(boolean brake) {
 		frontMotor.setBrake(brake);
-		backMotor.setBrake(brake);
 	}
 
 	protected void setTargetPositionMeters(double position) {
-		Rotation2d angleSetPoint = metersToMotorRotations(position);
-		frontMotor.applyAngleRequest(positionRequest.withSetPoint(angleSetPoint));
-		backMotor.applyAngleRequest(positionRequest.withSetPoint(angleSetPoint));
+		frontMotor.applyAngleRequest(positionRequest.withSetPoint(metersToMotorRotations(position)));
 	}
 
 	public boolean isAtBackwardLimit() {
 		return digitalInputsInputs.debouncedValue;
 	}
 
-	private Rotation2d getElevatorAngle() {
-		return Rotation2d.fromRotations(
-			(elevatorStuff.frontMotorStuff().positionSignal().getLatestValue().getRotations()
-				+ elevatorStuff.backMotorStuff().positionSignal().getLatestValue().getRotations()) / 2
-		);
-	}
-
 	public double getPositionMeters() {
-		return motorRotationsToMeters(getElevatorAngle());
+		return motorRotationsToMeters(elevatorStuff.motorStuff().positionSignal().getLatestValue());
 	}
 
 	protected void stayInPlace() {
 		setTargetPositionMeters(getPositionMeters());
 	}
 
-	private double motorRotationsToMeters(Rotation2d rotations) {
-		return rotations.getRotations() * ElevatorConstants.MOTOR_ROTATIONS_TO_METERS_CONVERSION_RATIO;
-	}
-
-	private Rotation2d metersToMotorRotations(double meters) {
-		return Rotation2d.fromRotations(meters / ElevatorConstants.MOTOR_ROTATIONS_TO_METERS_CONVERSION_RATIO);
-	}
-
 	protected void updateInputs() {
 		limitSwitch.updateInputs(digitalInputsInputs);
-		frontMotor.updateSignals(elevatorStuff.frontMotorStuff().positionSignal(), elevatorStuff.frontMotorStuff().voltageSignal());
-		backMotor.updateSignals(elevatorStuff.backMotorStuff().positionSignal(), elevatorStuff.backMotorStuff().voltageSignal());
+		frontMotor.updateSignals(elevatorStuff.motorStuff().positionSignal(), elevatorStuff.motorStuff().voltageSignal());
```

Comment:
Refactor to motor

Reply:
It's already merged,
Talk to yoav in whatsapp